CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ changes.
 
 - `hydra-node` logs will now report `NetworkEvents` to distinguish between `ConnectivityEvent`s and `ReceivedMessage`s on the network.
 
+- Hydra now uses a versioned protocol for handshaking. In the event of a node
+  attempting to connect using a different version of the networking protocol, a
+  `HandshakeFailure` event will be recorded in the logs.
+
 ## [0.17.0] - UNRELEASED
```

Comment:
Seems like the changelog got fragmented, let's merge it back again by removing this headline.